### PR TITLE
feat: new moblie member role card

### PR DESCRIPTION
**Pull request by liruchen32**
# Issue
[[會員制] 個人閱讀紀錄-會員頁UI更新](https://app.asana.com/0/1203699264568808/1205830825368922/f)

# Notice

# Dependency

### CODE REVIEW

File: packages/core/src/constants/reading-time-unit.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+export const READING_TIME_UNIT = {
```
Comment (Aylie-Chou) [inline packages/core/src/constants/reading-time-unit.js:1]: Could we add `freeze` to this const object (& also L7)

File: packages/react-components/src/mobile-member-role-card/index.js [hunk 2]
```diff
@@ -107,48 +155,73 @@ const MobileMemberRoleCard = ({
...
+        <InfoContainer>
+          {!hideInfo && (
+            <>
+              <TextContainer>
+                <P3Gray500 text={'閱讀篇數'} />
+                <Gray500BottomLine />
+                <P1TextContainer
+                  color={CardP1TextColor[role]}
+                  weight={P1.Weight.BOLD}
+                  text={articleReadCount.toLocaleString()}
```
Comment (Aylie-Chou) [inline packages/react-components/src/mobile-member-role-card/index.js:186]: same as [this comment](https://github.com/twreporter/twreporter-react/pull/2664/files#r1390631347)

File: packages/react-components/src/mobile-member-role-card/index.js [hunk 3]
```diff
@@ -107,48 +155,73 @@ const MobileMemberRoleCard = ({
   email = '',
   joinDate = '',
   name = '',
+  articleReadCount = 0,
```
Comment (Aylie-Chou) [inline packages/react-components/src/mobile-member-role-card/index.js:158]: Question: do we need to hide read count/time ui if `articleReadCount ` or `articleReadingTime ` is undefined ?

File: packages/react-components/src/mobile-member-role-card/stories/member-role-card.stories.js [hunk 4]
```diff
@@ -3,20 +3,28 @@ import MobileMemberRoleCard from '../index'
...
     releaseBranch: BRANCH_STORYBOOK_ARG_TYPE,
     role: getRadioArg(MEMBER_ROLE, MEMBER_ROLE.explorer),
+    articleReadingTimeUnit: getRadioArg(
+      READING_TIME_UNIT,
+      READING_TIME_UNIT.minute
+    ),
   },
 }
 
 export const moblieMemberRoleCard = args => <MobileMemberRoleCard {...args} />
```
Comment (Aylie-Chou) [inline packages/react-components/src/mobile-member-role-card/stories/member-role-card.stories.js:21]: This story would also need to be migrated to CSF3 format XD

File: packages/core/src/constants/reading-time-unit.js (hunk 1 above)
Comment (liruchen32) [reply packages/core/src/constants/reading-time-unit.js:1]: fixed

File: packages/react-components/src/mobile-member-role-card/index.js (hunk 3 above)
Comment (liruchen32) [reply packages/react-components/src/mobile-member-role-card/index.js:158]: 這應該是 defects 之類的

File: packages/react-components/src/mobile-member-role-card/stories/member-role-card.stories.js (hunk 4 above)
Comment (liruchen32) [reply packages/react-components/src/mobile-member-role-card/stories/member-role-card.stories.js:21]: 先等等

File: packages/react-components/src/mobile-member-role-card/index.js (hunk 3 above)
Comment (Aylie-Chou) [reply packages/react-components/src/mobile-member-role-card/index.js:158]: 蛤ＸＤ
那你再跟設計師討論吧

**Review by Aylie-Chou — APPROVED** [review]

**Pull request merged**